lib/machine-info.js

```diff
@@ -4,8 +4,12 @@ const writeFile = pify(fs.writeFile)
 const path = require('path')
 const _ = require('lodash/fp')
 
+let MACHINE_INFO = null

```

Comment:
Uppercase var names are usually reserved to consts.

Reply:
Also, since we're keeping this on a local variable, we can load from it as well. We can return early on the load if we already have something on the local variable.

Reply:
Fair point!